repository/copier.go

```diff
@@ -135,25 +153,42 @@ func (c *HDFSCopier) CopyToRemote(src, dst string, localFs billy.Filesystem) (er
 	}
 
 	// TODO to avoid this, we should implement a 'truncate' flag in 'client.Create' method
```

Comment:
does this comment still make sense here with the new content?

Reply:
The case is the same, overwriting a file if it already exists. Now it may make less sense as we do the check/delete dance twice. Once for the temporary file and another time for the destination file.

Reply:
Yes, yes, I mean the position. Maybe it's a bit confusing there. I'd move it before https://github.com/src-d/core-retrieval/pull/44/files#diff-71f2ce93b6503785a2eadc21b2d5cc29R181 that's when we're deleting (or manually truncating 😛) the file

Reply:
I believe that it is already in the correct place as the open+truncate would be made on the temporary file. The function never opens the destination file, it's renamed from temporary to destination.